mod/ping.php

```diff
@@ -357,9 +357,11 @@ function ping_get_notifications($uid) {
 			// Replace the name with {0} but ensure to make that only once
 			// The {0} is used later and prints the name in bold.
 
-			$pos = strpos($notification["message"],$notification['name']);
-			if ($pos !== false)
-				$notification["message"] = substr_replace($notification["message"],"{0}",$pos,strlen($notification["name"]));
+			if ( $notification['message'] ) {
```

Comment:
Could you please tweak it yourself ?
I understand pushing the button is convenient, but then maybe
add a pre-commit hook applying the style of preference ?